src/components/LocalNav.js

```diff
@@ -0,0 +1,27 @@
+import React, { Component } from 'react'
+
+import '../styles/local_nav.css'
+import '../../public/arrow.svg'
+
+class LocalNav extends Component {
+  render() {
+    return(
+      <nav className="local-nav">
+        <div className="local-nav-left"> Company Dasboard </div>
+        <div className="local-nav-right"></div>
+          <img className="logo-type" width="30" src="data:image/svg+xml;base64,PD94bWwgdmVyc2lvbj0iMS4wIiBlbmNvZGluZz0idXRmLTgiPz4NCjwhLS0gR2VuZXJhdG9yOiBBZG9iZSBJbGx1c3RyYXRvciAxNi4wLjAsIFNWRyBFeHBvcnQgUGx1Zy1JbiAuIFNWRyBWZXJzaW9uOiA2LjAwIEJ1aWxkIDApICAtLT4NCjwhRE9DVFlQRSBzdmcgUFVCTElDICItLy9XM0MvL0RURCBTVkcgMS4xLy9FTiIgImh0dHA6Ly93d3cudzMub3JnL0dyYXBoaWNzL1NWRy8xLjEvRFREL3N2ZzExLmR0ZCI+DQo8c3ZnIHZlcnNpb249IjEuMSIgaWQ9IkxheWVyXzEiIHhtbG5zPSJodHRwOi8vd3d3LnczLm9yZy8yMDAwL3N2ZyIgeG1sbnM6eGxpbms9Imh0dHA6Ly93d3cudzMub3JnLzE5OTkveGxpbmsiIHg9IjBweCIgeT0iMHB4Ig0KCSB3aWR0aD0iNTEycHgiIGhlaWdodD0iNTEycHgiIHZpZXdCb3g9IjAgMCA1MTIgNTEyIiBlbmFibGUtYmFja2dyb3VuZD0ibmV3IDAgMCA1MTIgNTEyIiB4bWw6c3BhY2U9InByZXNlcnZlIj4NCjxwYXRoIGZpbGw9IiNmZmZmZmYiIGQ9Ik0yNTYuMDMxLDI4OHYtNjMuNTYzYy02OC44MTMsMi4xMjUtMTI0LDU4LjIxOS0xMjQsMTI3LjU2M3M1NS4xODgsMTI1LjQwNiwxMjQsMTI3LjU5NFY0ODANCgljLTEwNi4wNjMsMC4wMzEtMTkyLjA2My04NS45MzgtMTkyLjA2My0xOTJzODYtMTkyLjAzMSwxOTIuMDYzLTE5MlYzMmwxOTIsMTI4TDI1Ni4wMzEsMjg4eiIvPg0KPC9zdmc+DQo=" role="presentation"/>
+          <a href="#" className="share">Share</a>
```

Comment:
Anchor tags will eventually need to become `<Link>`s. But we can save that for a future PR